sos/plugins/podman.py

```diff
@@ -74,24 +74,19 @@ def setup(self):
                 "podman network inspect %s" % net for net in nets
             ], subdir='networks')
 
-        ps_cmd = 'podman ps -q'
-        if self.get_option('all'):
-            ps_cmd = "%s -a" % ps_cmd
-
-        fmt = '{{lower .Repository}}:{{lower .Tag}} {{lower .ID}}'
-        img_cmd = "podman images --format='%s'" % fmt
-        vol_cmd = 'podman volume ls -q'
-
-        containers = self._get_podman_list(ps_cmd)
-        images = self._get_podman_list(img_cmd)
-        volumes = self._get_podman_list(vol_cmd)
+        containers = [
+            c[0] for c in self.get_containers(runtime='podman',
+                                              get_all=self.get_option('all'))
+        ]
+        images = self.get_container_images(runtime='podman')
+        volumes = self.get_container_volumes(runtime='podman')
 
         for container in containers:
             self.add_cmd_output("podman inspect %s" % container,
                                 subdir='containers')
 
         for img in images:
-            name, img_id = img.strip().split()
+            name, img_id = img
             insp = name if 'none' not in name else img_id
```

Comment:
This comment is not related to the change itself, but either way: what if a container name is e.g. `danone`, i.e. containing `none` string? I *think* it does not matter much, as `img_id` is always set (am I right here?). So the only impact would be less human-readable file `podman_inspect_<id>` will be stored instead of `podman_inspect_<name>`.

Reply:
There are situations where `podman` (and `docker` for that matter) will report an image name as a string of just 'none' or '<none>', so we could potentially make this a more exact match if needed. However, yes the image ID is always present, which is why we fallback like we do here.